docs/standard/library-guidance/breaking-changes.md

```diff
@@ -0,0 +1,98 @@
+
+  High-level libraries are directly referenced in an end-user application. If breaking changes occur the developer can choose to update to the latest version, or can modify their application to work with the breaking change.
+
+**✔️ DO** think about how your library will be used. What effect will breaking changes have on applications and libraries that use it?
+
+**✔️ DO** minimize breaking changes when developing a low-level .NET library.
+
+**✔️ CONSIDER** publishing a major rewrite of a library as a new NuGet package.
+
+## Types of breaking changes
+
+Breaking changes fall into different categories and aren't equally impactful.
+
+### Source breaking change
+
+A source breaking change doesn't affect program execution but will cause compilation errors the next time the application is recompiled. For example, a new overload can create an ambiguity in method calls that were unambiguous previously, or a renamed parameter will break callers that use named parameters.
+
+```csharp
+public class Task
+{
+    // Adding a type called Task could conflict with System.Threading.Tasks.Task at compilation
+}
+```
+
+Because a source breaking change is only harmful when the developer recompiles their application, it's the least disruptive breaking change. Developers can fix their own broken source code easily.
+
+### Behavior breaking change
+
+Behavior changes are the most common breaking change: almost any change in behavior could break someone. Changing the results a method produces, the exception your library throws, or the data output it writes to the network or disk could all negatively impact applications and libraries that depend on you. Even a bug fix can qualify as a breaking change if users relied on the previously broken behavior.
+
+Adding features and improving bad behaviors is a good thing, but without care it can make it very hard for existing software to upgrade. One approach to helping developers deal with behavior breaking changes is to hide them behind settings. Settings let developers update to the latest version of your library while at the same time choosing to opt in or opt out of breaking changes. This strategy lets developers stay up-to-date while letting their consuming code adapt over time.
+
+For example, ASP.NET Core MVC has the concept of a [compatibility version](/aspnet/core/mvc/compatibility-version) that modifies the features enabled and disabled on `MvcOptions`.
+
+**✔️ CONSIDER** leaving new features off by default if they affect existing users, and let developers opt in to the feature with a setting.
+
+### Binary breaking change
+
+A binary breaking change happens when you change the public API of a library so assemblies compiled against older versions are no longer able to call it. For example, changing a method's signature by adding a new parameter will cause assemblies compiled against the older version of the library to throw a <xref:System.MissingMethodException>.
```

Comment:
"_...you change **a** public API of **your** library so assemblies compiled against older versions **of your library** are no longer able to call **the API**._"